src/components/Thread/Thread.tsx

```diff
@@ -221,12 +221,86 @@ const ThreadInner = <
     }
   }, [threadMessages?.length]);
 
+  /** Keyboard Navigation */
```

Comment:
This code block seems quite similar to the change in `Channel`. Is there a way for this to be extracted in a separate piece (hook?)